packages/code-du-travail-utils/src/slugify.ts

```diff
@@ -1,14 +1,11 @@
-/**
- * @param {string} string
- */
-function slugify(string) {
+export function slugify(v: string) {
   const a =
     "àáäâãåăæąçćčđďèéěėëêęğǵḧìíïîįłḿǹńňñòóöôøṕŕřßşśšșťțùúüûǘůűūųẃẍÿýźžż·/_:;()";
   const b =
     "aaaaaaaaacccddeeeeeeegghiiiiilmnnnnoooooprrsssssttuuuuuuuuuwxyyzzz-------";
   const p = new RegExp(`[${a}]`, "g");
 
-  return string
+  return v
```

Comment:
J'ai pas l'impression que l'onn a améliorié le nomage ici 😄 
```suggestion
  return text
```